src/components/form/range/range.test.tsx

```diff
@@ -231,4 +231,53 @@ describe('EuiRange', () => {
       expect(container.firstChild).toMatchSnapshot();
     });
   });
+
+  describe('input aria-valuetext', () => {
+    const ticksWithLabels = [
+      {
+        label: '20kb',
+        value: 20,
+        accessibleLabel: 'twenty kilobytes',
+      },
+      {
+        label: '100kb',
+        value: 100,
+        accessibleLabel: 'one-hundred kilobytes',
+      },
+    ];
+
+    it('should exist when the current value has an accessible label', () => {
+      const { container } = render(
+        <EuiRange {...props} showTicks ticks={ticksWithLabels} value={20} />
+      );
+      const input = getByRole(container, 'slider');
+      expect(input.getAttribute('aria-valuetext')).toEqual(
+        '20, (twenty kilobytes)'
+      );
+    });
```

Comment:
We're not reusing `ticksWithLabels` more than once in the describe block so we should just inline it.
```suggestion
    it('should exist when the current value has an accessible label', () => {
      const { getByRole } = render(
        <EuiRange
          {...props}
          showTicks
          ticks={[
            {
              label: '20kb',
              value: 20,
              accessibleLabel: 'twenty kilobytes',
            },
            {
              label: '100kb',
              value: 100,
              accessibleLabel: 'one-hundred kilobytes',
            },
          ]}
          value={20}
        />
      );
      expect(getByRole('slider')).toHaveAttribute(
        'aria-valuetext',
        '20, (twenty kilobytes)'
      );
    });
```